tests/run-make/rustdoc-default-output/output-default.stdout

```diff
@@ -194,6 +194,8 @@ Options:
                         more information
         --html-no-source 
                         Disable HTML source code pages generation
+        --doctest-compilation-args add arguments to be used when compiling doctests
```

Comment:
This should document how argument splitting is handled, and what happens when the flag is set more than once.

Reply:
I added the explanation in the rustdoc book.

Reply:
"what happens when the flag is set more than once" does not seem to be explained?

Reply:
Good catch, gonna add it as well.